klaviyo/__init__.py

```diff
@@ -2,29 +2,41 @@
    from urllib.parse import urlencode
 except ImportError:
    from urllib import urlencode
+
 import base64
 import json
 import datetime
 import time
 
 import requests
 
-KLAVIYO_API_SERVER = 'a.klaviyo.com'
-KLAVIYO_TRACKING_ENDPOINT = 'api/track'
-KLAVIYO_IDENTIFY_ENDPOINT = 'api/identify'
+KLAVIYO_API_SERVER = 'https://a.klaviyo.com/api'
 KLAVIYO_DATA_VARIABLE = 'data'
-
+PUBLIC_TOKEN_REQUESTS = ('identify', 'track')
 TRACK_ONCE_KEY = '__track_once__'
 
+METRIC = 'metric'
+METRICS = 'metrics'
```

Comment:
I do not see the value of making them global instead of just having them inline where they are